src/main/java/ch/ejpd/lgs/searchindex/client/service/sync/FullSyncStateManager.java

```diff
@@ -177,25 +191,36 @@ public void incFullSeedMessageCounter() {
     currentFullSyncMessageCounter.getAndIncrement();
   }
 
-  private String validateOrDefaultSenderId(final String senderId) {
-    if (!isInMultiSenderMode && senderId == null) {
-      return singleSenderId;
+  public void incLandRegisterMessageCounter(String landRegister) {
+    if (Strings.isEmpty(landRegister)) {
+      return;
     }
-    if (senderId != null && validSenderIds.contains(senderId)) {
-      return senderId;
+
+    fullSyncMessagesTotalByLandRegister.putIfAbsent(landRegister, 0);
+    fullSyncMessagesTotalByLandRegister.compute(landRegister, (k, v) -> v + 1);
+  }
+
+  public void decrLandRegisterMessageCounter(Map<String, Integer> landRegisters) {
```

Comment:
don't use inc & decr abbreviations

Reply:
In this files all other methods in this class are using these abbreviations. Since the methods have similar logic I am using the same name convention to be somewhat unified